experimental/dimension/src/provider.rs

```diff
@@ -114,3 +117,55 @@ pub struct CurrencyPatterns {
     /// If the value is `None`, this means that the narrow pattern does not have a place holder.
     pub narrow_place_holder_index: Option<PlaceholderValue>,
 }
+
+#[icu_provider::data_struct(PercentEssentialsV1Marker = "percent/essentials@1")]
+#[derive(Default, Clone, PartialEq, Debug)]
+#[cfg_attr(
+    feature = "datagen",
+    derive(serde::Serialize, databake::Bake),
+    databake(path = icu_dimension::provider),
+)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+pub struct PercentEssentialsV1<'data> {
+    /// The index of the number placeholder in the standard pattern.
+    pub number_index: u16,
```

Comment:
do we expect patterns longer than 256?

Reply:
Likely never, updated to u8. Good catch :)